js/main.js

```diff
@@ -132,17 +151,33 @@ function getRecipeData(id) {
     }
     saveRecipeButton.addEventListener('click', function (event) {
       event.preventDefault();
-      var recipeId = selectedRecipePage.getAttribute('data-id');
-      var recipeImage = document.querySelector('.selected-recipe-image');
-      var recipeInfo = {
-        recipeID: recipeId,
-        image: recipeImage.getAttribute('src'),
-        title: $header.textContent
-      };
-      data.recipes.push(recipeInfo);
-      var newSavedRecipe = appendSavedRecipe(recipeInfo);
-      savedRecipePage.appendChild(newSavedRecipe);
-    });
+      if (saveRecipeButton.textContent === 'SAVE RECIPE') {
+        var recipeId = selectedRecipePage.getAttribute('data-id');
+        var recipeImage = document.querySelector('.selected-recipe-image');
+        var recipeInfo = {
+          recipeID: recipeId,
+          image: recipeImage.getAttribute('src'),
+          title: $header.textContent
+        };
+        data.recipes.push(recipeInfo);
+        var newSavedRecipe = appendSavedRecipe(recipeInfo);
+        savedRecipePage.appendChild(newSavedRecipe);
+      } else if (saveRecipeButton.textContent === 'REMOVE RECIPE') {
```

Comment:
I wouldn't recommend basing this logic on text content, that could get changed at any time. Instead, you should work off of a class or data- attribute applied to the element